metrics-scripts/encapsulate_metrics_script.py

```diff
@@ -62,20 +107,20 @@ def calculate_block_rates():
                         second_last_height, second_last_height_ts = last_height, last_height_ts
                     last_height, last_height_ts = height, ts
 
-    round_rate = None
-    height_rate = None
+    round_time = None
+    height_time = None
 
     if second_last_round is not None and last_round != second_last_round:
-        round_rate = (last_round_ts - second_last_round_ts) / 1e9
+        round_time = (last_round_ts - second_last_round_ts) / 1e9
 
     if second_last_height is not None and last_height != second_last_height:
-        height_rate = (last_height_ts - second_last_height_ts) / 1e9
+        height_time = (last_height_ts - second_last_height_ts) / 1e9
```

Comment:
I dont think calculating block time or round time metrics would particularly useful for monitoring, lets discuss this today in the call